README.md

```diff
@@ -1,2 +1,77 @@
-# javascript-capstone
-javascript-capstone
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# MealzDB Web Application.
+
+- This is a web application that allows you to get information about various kinds of meals and recipes from all over the world.
+- Using, MealzDB you can get information about various meals and recipes from all over the world.
+- Using Interactive API You can make comments live and make reservations for meals. Also you can even like meals and add them to your faviourites. :heart_eyes:
+
+## Built With
+
+- **Webpack** :gem:
+- HTML SCSS & VANILLA JAVASCRIPT
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+> ## [Live Video](https://drive.google.com/file/d/1dOBHpVDgSMTmSKXqu62OfDP_OPVLCxIG/view?usp=sharing)
+
+### Install
+
+- To Install First Either downlaod the zip files or clone the repository directly from the code buttons.
+  Then run the following command:
+  `npx install `
+  Then run the following command
+  ` npm Start`
```

Comment:
[OPTIONAL] `npm` commands are case sensitive, it would be better to use `npm start`.